### ISSUE TITLE
A11y: Invoking "Ongoing tasks" button does not read out of opened flyout

### ISSUE THREAD

**Issue by chingucoding** [files-community/Files#2577]
**Describe the bug**
When invoking the ongoing tasks button, users are not notified about the content of the opened flyout. Generally, when invoking a button, users should be notified of changes, e.g. reading out something or switching focus if applicable

**To Reproduce**
1. Visit a folder
2. Invoke ongoing task button
3. Observe nothing gets announced

**Expected behavior**
Ideally, narrator should announce the changes. That is something that the WinUI InfoBar control already does: https://github.com/microsoft/microsoft-ui-xaml/blob/50f29e79da9b457d9b136293d82ff21b8340bf2a/dev/InfoBar/InfoBarAutomationPeer.cpp#L24

So in the case of Files, you should raise a [notification event](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.automationpeer.raisenotificationevent?view=winrt-19041) with [processingkind](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.automationnotificationprocessing?view=winrt-19041) ImportantMostRecent.

**Screenshots**

**Desktop (please complete the following information):**
- OS: Insider Build 10.0.20262 Build 20262
- App version: 0.21.1.0

**Comment by yaichenbaum** [comment]
@chingucoding This issue should be resolved with #3312.

**Comment by marcelwgn** [comment]
Thank you @winston-de and @yair100 for fixing this!
